contrib/epee/src/abstract_http_client.cpp

```diff
@@ -26,19 +26,15 @@ namespace net_utils
   //----------------------------------------------------------------------------------------------------
   std::string dec_to_hex(char num, int radix)
```

Comment:
This whole function is an abomination of the common sense (not related to your changes though). Why call it `dec_to_hex` and then supply `radix` argument? If it's hex only, remove the argument and rewrite it with just a few lines of code to make 2 hex characters out of a single-byte `char` value. Using `std::string` and `std::reverse` here is a cherry on the top, why? To make it 10 times slower than needed with some memory allocation? Example: http://cpp.sh/9xt2hk

Reply:
Just checked: it's used only in 1 place and only with radix = 16, so it makes sense to rewrite it completely